src/library_sockfs.js

```diff
@@ -619,16 +619,13 @@ addToLibrary({
           buffer = buffer.buffer;
         }
 
-        var data;
-#if PTHREADS
-        // WebSockets .send() does not allow passing a SharedArrayBuffer, so clone the portion of the SharedArrayBuffer as a regular
-        // ArrayBuffer that we want to send.
-        if (buffer instanceof SharedArrayBuffer) {
-          data = new Uint8Array(new Uint8Array(buffer.slice(offset, offset + length))).buffer;
-        } else {
-#endif
-          data = buffer.slice(offset, offset + length);
+        var data = buffer.slice(offset, offset + length);
 #if PTHREADS
```

Comment:
Separately, should this be `#if SHARED_MEMORY` so it works with `WASM_WORKERS`?

Reply:
Agreed... maybe a followup?

Reply:
Sounds good.